Sources/TerminalAPIKit/Models/Enumerations/SessionType.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  SessionType.swift
+//  TerminalAPIKit
+//
+//  Created by Andrew Gates on 8/8/23.
+//
+
+import Foundation
+
+public enum SessionType: String, Codable {
+    case begin = "Begin"
```

Comment:
can we add comments to the type, and every case?
this applies to multiple places in the PR

Reply:
I searched 125 results of `String, CodingKey` and 54 results of `String, Codable` and none of them have comments. What are you looking for here, since I have no examples to go off of, and I'll be breaking the trend with the rest of the code base?